### DISCUSSION TITLE
Build a standalone app

### DISCUSSION THREAD

**Discussion by jean-smaug** [adonisjs/core#3853] [Help]
👋 Hello,

I've got a project with the following structure 👇 

```
|-- packages
|---- package-1
|---- package-2
|-- services
|---- my-adonis-app
```

The project is managed by PNPM and I'm using the `workspace:VERSION` to reference packages into the adonis app.

Is it possible to create an Adonis standalone build that contains the app and all dependencies that are necessaries to run it.
Create a server bundle in short.
Base on what I read in the doc, Encore seems to be for client only.

Thanks

**Comment by Barbapapazes** [comment]
To my knowledge, it's not included.
